### ISSUE TITLE
UnusedParameterSniff does not see static method call with a variable

### ISSUE THREAD

**Issue by ondrejmirtes** [slevomat/coding-standard#434]
```php
private static function resolvePhpDocBlockFromClass(
	string $resolveMethodName
): ?self
{
	return self::$resolveMethodName();
}
```

`$resolveMethodName` parameter is reported as unused here.

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/d51e9318104ba855dd366e9a9463b1fd3044143f
